webpack.config.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
 			encoding: 'utf8',
 		}),
 	],
+	node: {
+		fs: 'empty',
```

Comment:
can you leave a comment explaining why this needs to be here?

Reply:
I added that without doing much investigation but it will suppress the following error `Module not found: Error: Can't resolve 'fs'` while I still using `css-loader@3.2.1`
<img width="1308" alt="Screenshot 2020-02-14 at 17 46 43" src="https://user-images.githubusercontent.com/9718606/74554670-fd5e9180-4f51-11ea-9492-be75d2d05dd0.png">
shall I create a ticket to investigate that?

Reply:
this is resolved now with the refactoring done